### ISSUE TITLE
Unable to add a third head node to VM cluster

### ISSUE THREAD

**Issue by amithkanand** [bloomberg/chef-bach#774]
I am seeing following errors when I add third head node to the cluster with `BCPC-Hadoop-Head` role assigned to it.

```
================================================================================
    Error executing action `run` on resource 'ruby_block[setup-jn-data]'
    ================================================================================

    NoMethodError
    -------------
    undefined method `bash' for Chef::Resource::RubyBlock

    Cookbook Trace:
    ---------------
    /var/chef/cache/cookbooks/bcpc-hadoop/recipes/journalnode.rb:52:in `block (2 levels) in from_file'

    Resource Declaration:
    ---------------------
    # In /var/chef/cache/cookbooks/bcpc-hadoop/recipes/journalnode.rb
```

and

```
Recipe: <Dynamically Defined Resource>
    * directory[/disk/1/dfs/jn/] action create (up to date)
    * directory[/disk/1/dfs/jn/Test-Laptop] action create (up to date)
    * file[/var/chef/cache/jn_fmt.tgz] action create (up to date)

    ================================================================================
    Error executing action `run` on resource 'ruby_block[setup-jn-data]'
    ================================================================================

    NoMethodError
    -------------
    undefined method `resolve_notification_resources' for Chef::Resource::RubyBlock

    Cookbook Trace:
    ---------------
    /var/chef/cache/cookbooks/bcpc-hadoop/recipes/journalnode.rb:65:in `block (2 levels) in from_file'

    Resource Declaration:
    ---------------------
    # In /var/chef/cache/cookbooks/bcpc-hadoop/recipes/journalnode.rb
```

**Comment by http-418** [comment]
Fixed in PR #772

**Comment by vt0r** [comment]
Resolved when #772 was merged.
